### PR TITLE
clarified sendeventbatchasync docs

### PR DESCRIPTION
**Pull request by gregman-msft**
docs:clarified SendEventBatchAsync requires AMQP over WebSockets.

### CODE REVIEW

File: iothub/device/src/DeviceClient.cs [hunk 1]
```diff
@@ -394,14 +394,14 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
 
 
         /// <summary>
-        /// Sends an event to device hub
+        /// Sends an event to device hub. Requires AMQP over WebSockets.
         /// </summary>
```
Comment (prmathur-microsoft) [inline iothub/device/src/DeviceClient.cs]: This is generic one which works on all protocols. Batching is the one that works only on AMQP. Please revert this.

File: iothub/device/src/DeviceClient.cs [hunk 2]
```diff
@@ -394,14 +394,14 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
...
         /// <param name="message">The message.</param>
         /// <returns>The message containing the event</returns>
         public Task SendEventAsync(Message message) => this.internalClient.SendEventAsync(message);
 
         /// <summary>
-        /// Sends an event to device hub
+        /// Sends an event to device hub. Requires AMQP over WebSockets.
         /// </summary>
```
Comment (prmathur-microsoft) [inline iothub/device/src/DeviceClient.cs]: Same comment as above.

File: iothub/device/src/ModuleClient.cs [hunk 3]
```diff
@@ -349,14 +349,14 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         public Task SendEventAsync(Message message, CancellationToken cancellationToken) => this.internalClient.SendEventAsync(message, cancellationToken);
 
         /// <summary>
-        /// Sends a batch of events to device hub
+        /// Sends a batch of events to device hub. Requires AMQP over WebSockets.
         /// </summary>
```
Comment (prmathur-microsoft) [inline iothub/device/src/ModuleClient.cs]: AMQP or AMQP over websocket.

@abhipsaMisra - Do we have E2E for this? If not can you add?.

Comment (abhipsaMisra) [reply iothub/device/src/ModuleClient.cs]: Currently we don't have any E2E for SendEventBatchAsync. Tracked in backlog.

**Pull request merged**